newickFormatReader.py

```diff
@@ -15,12 +15,23 @@
 # of strings of the form:
 # (topVertex, bottomVertex, leftEdgeName, rightEdgeName)
 
-# python librarieså
+# Python libraries
 from io import StringIO
 
 # BioPython libraries
 from Bio import Phylo
 
+# Storage class for the newick data (trees, tip mapping, and optional distance parameters)
```

Comment:
I think this two line comment should be change to a block comment under the class definition:
```python
class ReconData(object):
    """
    Storage class for for the newick data ...
    Distances encode the annotated ...
    """
   def __init__(...):
```
This will allow automatic documentation generation in the future.